### ISSUE TITLE
Decide on `scheduled-daily`

### ISSUE THREAD

**Issue by stappersg** [canonical/cloud-init#5929]
Please make a decision on the _github action_ `scheduled-daily`.

Either fix the _daily scheduled CI actions_ or simply disable them.

Current state seems to be _"we know it is broken and we don't care"_ (OK, _"we know it is broken and everything else in the cloud-init project is way more important"_). The signal of _we know it is broken_ is harmful for this cool project (emitting _we don't care_ is **not** cool).

**Comment by TheRealFalcon** [comment]
@stappersg , is this negatively affecting you? If so, can you explain how?

Most of our daily actions exist precisely because we don't want them to be "voting" actions for per-PR CI, but their failure still provides a signal that we'll want to fix eventually. Why do you see this as a problem?

**Comment by stappersg** [comment]
On Mon, Dec 16, 2024 at 01:52:00PM -0800, James Falcon wrote:
> @stappersg , is this negatively affecting you?

} @stappersg , is this negatively affecting you? [Y/N]

Yes


> If so, can you explain how?

An attempt follows

 
> Most of our daily actions exist precisely because we don't want them
> to be "voting" actions for per-PR CI, but their failure still provides
> a signal that we'll want to fix eventually.
> Why do you see this as a problem?

The problem I see is  "We don't care".  That indifference hurts me.

I, a sysadmin, do use cloud-init.  I want to know it better.
So did "a github.com fork". Got since that day every day build failure
from github CI service. (Most likely all other "forkers" get those daily
build failure messages.)

After seeing the build failures long enough
I filed https://github.com/canonical/cloud-init/issues/3620#issuecomment-2471560129

Follow-up was https://github.com/canonical/cloud-init/pull/5867 (merged)

Next build failure got exposed. I examed / research it,
had to file https://github.com/canonical/cloud-init/issues/5869
It got marked "good first issue".  (And this part-time Python programmer
got scared.)

Seen several more days of build failures from "scheduled daily".
Feeling bad of bailing out to silence github action, tried a different
approach with https://github.com/canonical/cloud-init/pull/5884
If merged it would have helped all other "forkers".  But the merge
request got rejected with "should be a straightforward fix".
That was on 2024-11-18.

A fat three weeks of daily reminders on build failures
from `scheduled-daily` was this issue opent.

Here we are now.

Please decide on `scheduled-daily`. Both "disable" and "fix" stop
emitting "All other things in cloud-init project are more as
good daily routine".


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by aciba90** [comment]
As the proposal for disabling `scheduled-daily` is not wanted by the team and the lack of activity here, I proceed to close this, thanks!
